src/azure-cli-testsdk/setup.py

```diff
@@ -34,7 +34,7 @@
     'jmespath',
     'mock',
     'vcrpy>=1.10.3',
-    'azure-devtools==0.5.2'
+    'azure-devtools>=0.5.4'
```

Comment:
Once the PR is approved, I shall ask to publish out a new azure-devtools containing https://github.com/Azure/azure-python-devtools/pull/40, and remove the private wheel from this PR